core/scss/components/badge.scss

```diff
@@ -13,6 +13,12 @@
     @include badge-size;
     @include badge-style($badge--gray__background-color, $radius: true);
 
+    &--outline {
```

Comment:
Should this be added to the mixin instead? @bfbiggs

Reply:
@pauljeter now using a mixin; looks like there was already one there, I just overlooked it previously